src/components/molecules/Sidebar/Sidebar.scss

```diff
@@ -9,7 +9,7 @@ $dark: #19181a;
   z-index: z(sidebar);
   right: 0;
   top: 60px;
-  width: 360px;
+  min-width: 300px;
```

Comment:
Why the change from `360px` to `300px`? What happens when the window is super-wide? Should we have a max-width as well?